nmp/all.nmp

```diff
@@ -2783,30 +2733,26 @@ op xpwrite(src: LOC16)
 		// Prog[0x3F:AL] = [loc16];
 	}
 
-// DOC: page 467
-op xret()
-	syntax = "XRET"
-	image = "0101 0110 1111 1111"
+// DOC: page 467, 468
+op xret__xretc_cond(c: COND)
+	syntax = if c.num == 0b1111 then "XRET" else format("XRETC %s", c) endif
```

Comment:
This could be a general issue, in which case we need to change `COND`. Can you determine whether `			case 0b1111: "UNC" // Unconditional –` ever gets displayed as `UNC`, or it's always blank? If it's blank you need to change it to blank in tms.nmp

Reply:
This is the output of `dis2000` : 
```
00000002   56ff   XRETC        UNC
```
and as said in the DOC (page 467): 

> XRET is an alias for XRETC unconditional.

Reply:
Ah, so shouldn't we output `XRETC UNC` instead of `XRET` @Alex-Ralph ?